### PR TITLE
Move setup back into README.md

### PR DESCRIPTION
**Pull request by DerekRoberts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,3 +196,316 @@ Local development can be supported using Docker Compose.  Please be aware that P
...
+
+Manage Action Secrets from your Repo > Settings > Secrets > Actions.
+
+
+#### Required
+
+**GHTOKEN**
+
+* Default token, not viewable, common to all repositories
+* Variable: `{{ secrets.GHTOKEN }}`
```
Comment (mishraomp) [inline README.md:296]: is this one same as `GITHUB_TOKEN`

**Review by mishraomp — COMMENTED** [review]
just one quick question, everything else looks good.

File: README.md (hunk 1 above)
Comment (DerekRoberts) [reply README.md]: That's correct.  I cleaned up the token, but don't want to do much else.  The setup section should be getting overhauled this sprint.

**Review by mishraomp — APPROVED** [review]
CITY

**Pull request merged**